docs/src/pages/components/autocomplete/IntegrationDownshift.tsx

```diff
@@ -171,46 +171,54 @@ function DownshiftMultiple(props: DownshiftMultipleProps) {
       {({
         getInputProps,
         getItemProps,
+        getLabelProps,
         isOpen,
         inputValue: inputValue2,
         selectedItem: selectedItem2,
         highlightedIndex,
-      }) => (
-        <div className={classes.container}>
-          {renderInput({
-            fullWidth: true,
-            classes,
-            InputProps: getInputProps({
-              startAdornment: selectedItem.map(item => (
-                <Chip
-                  key={item}
-                  tabIndex={-1}
-                  label={item}
-                  className={classes.chip}
-                  onDelete={handleDelete(item)}
-                />
-              )),
-              onChange: handleInputChange,
-              onKeyDown: handleKeyDown,
-              placeholder: 'Select multiple countries',
-            }),
-            label: 'Label',
-          })}
-          {isOpen ? (
-            <Paper className={classes.paper} square>
-              {getSuggestions(inputValue2!).map((suggestion, index) =>
-                renderSuggestion({
-                  suggestion,
-                  index,
-                  itemProps: getItemProps({ item: suggestion.label }),
-                  highlightedIndex,
-                  selectedItem: selectedItem2,
-                }),
-              )}
-            </Paper>
-          ) : null}
-        </div>
-      )}
+      }) => {
+        const { onBlur, onFocus, ...inputProps } = getInputProps();
+        return (
+          <div className={classes.container}>
+            {renderInput({
+              fullWidth: true,
+              classes,
+              label: 'countries',
+              InputLabelProps: getLabelProps({ strink: true } as any),
```

Comment:
```suggestion
              InputLabelProps: getLabelProps({ shrink: true } as any),
```

Reply:
It's not needed, we can remove the prop

Reply:
No need to be snarky. Seems like I forgot to include some suggestions.